src/layout/ShowRawButton.tsx

```diff
@@ -11,13 +11,21 @@ const ShowRawButton = ({ obj, ...props }: { obj: object } & Omit<ButtonProps, 'o
       <Button {...buttonProps} {...props}>
         <Trans i18nKey={'raw_content'}>Raw content</Trans>
       </Button>
-      <Box {...disclosureProps} wordBreak='break-all' whiteSpace='pre' color='white' borderRadius='md' maxW={'90vw'}>
-        <Code w={'full'} wordBreak='break-all' p='4' fontFamily='monospace' overflowX='auto'>
-          {JSON.stringify(obj, null, 2)}
-        </Code>
+      <Box {...disclosureProps}>
+        <RawContentBox obj={obj} />
       </Box>
     </>
   )
 }
 
+export const RawContentBox = ({ obj, ...rest }: { obj: object } & BoxProps) => {
+  return (
+    <Box wordBreak='break-all' whiteSpace='pre' color='white' borderRadius='md' maxW={'90vw'} {...rest}>
+      <Code w={'full'} wordBreak='break-all' p='4' fontFamily='monospace' overflowX='auto'>
+        {JSON.stringify(obj, null, 2)}
+      </Code>
+    </Box>
+  )
+}
```

Comment:
I've seen this in many components in this PR. Just as a friendly reminder, there's no need to `return` when not using hooks or other initialization before the actual components:
```suggestion
export const RawContentBox = ({ obj, ...rest }: { obj: object } & BoxProps) => (
  <Box wordBreak='break-all' whiteSpace='pre' color='white' borderRadius='md' maxW={'90vw'} {...rest}>
    <Code w={'full'} wordBreak='break-all' p='4' fontFamily='monospace' overflowX='auto'>
      {JSON.stringify(obj, null, 2)}
    </Code>
  </Box>
)
```

Reply:
I add the return just for improve readability, i know about it can be avoided. But if you prefer without return keyword i will do it like that from now

Reply:
1de0206